src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -119,12 +119,9 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
   const {travellerSelectionMode, zoneSelectionMode} =
     fareProductTypeConfig.configuration;
 
-  const offerEndpoint =
-    zoneSelectionMode === 'none'
-      ? 'authority'
-      : zoneSelectionMode === 'multiple-stop-harbor'
-      ? 'stop-places'
-      : 'zones';
+  const offerEndpoint = params.fareProductTypeConfig.configuration.offerEndpoint
+    ? params.fareProductTypeConfig.configuration.offerEndpoint
+    : 'zones';
```

Comment:
Let's just make offerEndpoint required in Firestore, and skip this fallback.

Reply:
Now updated in app